code/modules/cargo/exports/sheets.dm

```diff
@@ -0,0 +1,123 @@
+//
+// Sheet Exports
+//
+
+/datum/export/stack
+	unit_name = "sheet"
+
+/datum/export/stack/get_amount(obj/O)
+	var/obj/item/stack/S = O
+	if(istype(S))
+		return S.amount
+	return 0
+
+
+// Leather, skin and other farming by-products.
+
+/datum/export/stack/skin
+	unit_name = ""
+
+// Monkey hide. Cheap.
+/datum/export/stack/skin/monkey
+	cost = 150
+	unit_name = "monkey hide"
+	export_types = list(/obj/item/stack/sheet/animalhide/monkey)
+
+// Human skin. Illegal
+/datum/export/stack/skin/human
```

Comment:
It _seems_. You need a gibber (chief), some human bodies (dead players, or monkey cubes and help from geneticist/chemist) and a lot of patience. And what's your profit in the end? 1 human skin = 4 plasma sheets = 4 crates = 4 fuel tanks. Not that much.